src/components/quantityLegend.js

```diff
@@ -0,0 +1,66 @@
+import React from "react"
+
+const QuantityLegend = ({
+  text,
+  boxSize,
+  imgSrc,
+  background,
+  quantity,
+  isPercent,
+  invertedBoldText,
+  wrapStyle,
+  boxStyle,
+  descTextStyle,
+}) => {
+  const quantityDisplayText = () => (isPercent ? `${quantity}%` : `${quantity}`)
```

Comment:
I'm not an expert in React but I think assigning to constant directly also work since react will run the whole component function when the prop or state is changed.

```js
const quantityDisplayText = isPercent ? `${quantity}%` : `${quantity}`
```

This will make the code easier to read. Or is there any factor that I missed?